### PR TITLE
InCanvasSearch: scrolling fix

### PR DESCRIPTION
**Pull request by aosyatnik**
### Purpose

Link to YouTrack:
[MAGN-7522](http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-7522) Should be able to use mouse wheel to scroll in-canvas search

Reason of bug:
Horizontal scrollviewer(red arrow) handles all mouse wheel events.
image

To scroll vertical scrollviewer we have to change vertical offset manually.

### Declarations

Check these if you believe they are true

- [x] The code base is in a better state after this PR
- [x] The level of testing this PR includes is appropriate

### Reviewers

@ramramps 
@pboyer 
@Benglin 

### FYIs

@lillismith

### CODE REVIEW

File: src/DynamoCoreWpf/Controls/IncanvasSearchControl.xaml [hunk 1]
```diff
@@ -106,29 +112,24 @@
...
+                     ItemContainerStyle="{StaticResource SearchMemberStyle}"
+                     ItemsSource="{Binding SearchResults, NotifyOnTargetUpdated=True}"
+                     TargetUpdated="OnMembersListBoxUpdated"
+                     PreviewMouseWheel="OnMembersListBoxMouseWheel">
+                <ListBox.ItemTemplate>
+                    <DataTemplate>
+                        <ScrollViewer Template="{StaticResource InCanvasScrollViewerControlTemplate}"
+                                      Style="{StaticResource InCanvasScrollViewerStyle}"
+                                      CanContentScroll="True"
+                                      Width="250">
```
Comment (Benglin) [inline src/DynamoCoreWpf/Controls/IncanvasSearchControl.xaml]: Should the `ScrollViewer` width be tied to the `MembersListBox.Width`? Or is this by design?

Comment (aosyatnik) [inline src/DynamoCoreWpf/Controls/IncanvasSearchControl.xaml]: This width should be the same as InCanvasSearch width, which is 250.

File: src/DynamoCoreWpf/Controls/IncanvasSearchControl.xaml.cs [hunk 2]
```diff
@@ -216,5 +222,16 @@ private void OnInCanvasSearchKeyDown(object sender, KeyEventArgs e)
 
             return nextselectedMemberIndex;
         }
+
+        private void OnMembersListBoxMouseWheel(object sender, MouseWheelEventArgs e)
+        {
+            var listBox = sender as FrameworkElement;
+
+            var scrollViewer = listBox.ChildOfType<ScrollViewer>();
```
Comment (Benglin) [inline src/DynamoCoreWpf/Controls/IncanvasSearchControl.xaml.cs]: There's no `null` checks for both `listBox` and `scrollViewer`, can you add those? Thanks!

**Comment by ikeough** [comment]
@aosyatnik is the horizontal scroll control really below the first result? I think it should be at the bottom of the results. @lillismith can you comment here? 

Also, I think generally the sizing in the in-canvas search is off. Horizontal scroll bars are painful to use and we're adding them because we're putting a ton of text on every line. All the cut off results make it look like we're not properly sizing our UI.

I would like to propose several options:

- Don't show the method signature (i.e. show Point.ByCoordinates, not Point.ByCoordinates(x:double = 0,...)). In this case we would rely on the popup tooltip to differentiate between the results.
- Put the method signature in a smaller font, below the name of the method.
- Make the search results box re-size its width to accommodate everything. We've got the whole canvas to play with. Why are we only using 250px?

**Comment by lillismith** [comment]
These are great suggestions, but are a little outside the scope of this bug which is about getting scroll wheel to work on in-canvas search results. Let's open a separate story and we can work on tuning in-canvas search up to look better incrementally.

**Comment by pboyer** [comment]
@lillismith I just want to make sure we've added the suggested tasks.

**Pull request merged**